src/shared/services/i18n/fallback/locales/nl/translations.json

```diff
@@ -0,0 +1,63 @@
+{
+    "algemeen": {
+        "toestemming-delen-contactgegevens": "Ja, ik geef Signalen toestemming om mijn contactgegevens te delen met andere organisaties, als dat nodig is om mijn melding goed op te lossen.",
+        "wijk": "Wijk",
+        "kop-titel": "",
```

Comment:
Do we need this?

Reply:
I guess. It might be used by other municipalities